tests/happy-path.js

```diff
@@ -74,6 +78,12 @@ describe('Happy Path', () => {
     assert.isTrue(await isEnvRunning(cwd));
   });
 
+  it('env --info', async () => {
+    const res = await exec('aeproject env --info', { cwd });
+    assert.equal(res.code, 0);
+    print(res.stdout)
```

Comment:
why print? should not assert be enough?

Reply:
just to confirm/see in ci what the actually used version is